src/api/controllers/articleController.js

```diff
@@ -3,9 +3,10 @@ import models from '../models';
 import statusCodes from '../../helpers/constants/status.codes';
 import errorMessage from '../../helpers/constants/error.messages';
 import articleValidator from '../../helpers/validators/articleValidator';
+import sendError from '../../helpers/error.sender';
```

Comment:
can we switch to ``errorSender``? I think its more logical

Reply:
the file is called errorSender but I the function is called sendError